### ISSUE TITLE
[wishlist] Use colorpicker instead of colortable

### ISSUE THREAD

**Issue by ltog** [umap-project/umap#411]
When choosing a color for a marker, instead of

![no-color-picker](https://cloud.githubusercontent.com/assets/4726277/20685214/cb953de2-b5b3-11e6-9821-9d9c122b7932.png)

I'd like to see a colorpicker, such as the one from GIMP:

![colorpicker-gimp](https://cloud.githubusercontent.com/assets/4726277/20685226/de16c954-b5b3-11e6-96a8-df79df634a8f.png)

**Comment by grandbic** [comment]
I'd like to see a different colour chooser, as well!

**Comment by yohanboniface** [comment]
Yes, would be better, BUT we cannot just switch to `<input type=color` because we also need to be able to write a variable inside the input, like `{color}` or any property, and in this case uMap will read this property value and use it as color.
